components/ChatWindow.tsx

```diff
@@ -173,11 +188,16 @@ export function ChatWindow(props: {
       worker.current = new Worker(new URL('../app/worker.ts', import.meta.url), {
         type: 'module',
       });
+      worker.current.postMessage('caca')
```

Comment:
This is to kick off the client initialization I assume?

Could we have some special event that initializes the client, then waits for a response from the worker before continuing?

Reply:
that was just a debug leftover